clj/build.boot

```diff
@@ -51,6 +62,16 @@
            (install)
            (target)))
 
+(deftask build-cljs
+  "build clojurescript"
+  []
+  (comp
+   (test-cljs)
+   (cljs :optimizations :advanced)))
+
+(deftask build []
+  (comp (build-cljs) (build-clj)))
```

Comment:
We'd also need to switch the order of `(test-cljs)` and `(cljs. ...)` because the compilation is also not run.